src/components/CountrySelect/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import './style.css';
+import { CountryDropdown, RegionDropdown } from 'react-country-region-selector';
+import axios from 'axios';
+
+class CountrySelect extends Component {
+  constructor (props) {
+    super(props);
+    this.state = { 
+      place: {
+        country: 'Armenia', 
+        region: 'Yerevan',
+      },
+      teperature: '',
+      desc: '',
+    };
+  }
+
+  selectCountry (val) {
+    this.setState({ country: val });
+  }
+
+  selectRegion (val) {
+    this.setState({ region: val });
+  }
+
+  componentDidMount(){
+    this.getWeather(this.state.region)
+  }
+
+  getWeather(region){
+    // console.log(region);
+    // console.log(window)
+
+    axios({
+      method: "GET",
+      url: `https://api.openweathermap.org/data/2.5/weather?q=${region}&appid=2aeac406d27d1caf020cdd5e447d1bab`,
+    })
```

Comment:
Store URL, API key in a different file import from there. ( file name can be config.js ).
```js
import { getOpenweatherUrl } from '../config.js'
 ...
const url = getOpenweatherUrl(region);
axios({ method: "GET", url }).then(...
```